### PR TITLE
Move demo, add pnpm hash update script, Mention license on the bottom right 

### PR DESCRIPTION
**Pull request by SuperSandro2000**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -15,6 +15,7 @@ <h1>{{title}}</h1>
 </main>
 
 <footer class="typography">
-  <p>NuschtOS Search</p>
+  <p>NüschtOS Search</p>
+  <p>Licensed under MIT</p>
```
Comment (MarcelCoding) [inline src/app/app.component.html:19]: I would like to put this behind the source link and in brackets.

Comment (SuperSandro2000) [reply src/app/app.component.html:19]: You mean I should switch this line with the next and WDYM with brackets?

File: src/app/app.component.html [hunk 2]
```diff
@@ -15,6 +15,7 @@ <h1>{{title}}</h1>
...
   <a href="https://github.com/NuschtOS/search" target="_blank">Source</a>
```
Comment (SuperSandro2000) [inline src/app/app.component.html:20]:
```suggestion
  <a href="https://github.com/NuschtOS/search" target="_blank">Source</a>
  <p>(Licensed under MIT)</p>
```

**Review by MarcelCoding — APPROVED** [review]
I'll fix the pipeline

File: .github/workflows/gh-pages.yaml [hunk 3]
```diff
@@ -32,12 +32,7 @@ jobs:
 
       - name: Build search
         working-directory: demo
```
Comment (SuperSandro2000) [inline .github/workflows/gh-pages.yaml:34]:
```suggestion
```

**Comment by SuperSandro2000** [comment]
@MarcelCoding that is not working as intended

File: .github/workflows/gh-pages.yaml [hunk 4]
```diff
@@ -31,29 +31,13 @@ jobs:
...
 
       - name: Build search
-        working-directory: demo
-        run: nix build -L --override-input search ..
-
-      - name: Upload artifact
-        uses: actions/upload-pages-artifact@v3
-        with:
-          path: demo/result
+        run: nix build -L --override-input search . github:NuschtOS/search.nuschtos.de
```
Comment (SuperSandro2000) [inline .github/workflows/gh-pages.yaml:34]:
```suggestion
        run: nix build -L github:NuschtOS/search.nuschtos.de --override-input search .
```

**Pull request merged**